rmgpy/pdep/collision.pyx

```diff
@@ -31,15 +31,16 @@
 This module contains classes and functions for working with collision models.
 """
 
-cimport cython
-from libc.math cimport exp, sqrt
+from libc.math cimport exp
```

Comment:
I know I changed this, but could you move the `libc` import to after numpy? Also in configuration.pyx, cse.pyx, and msc.pyx, and reaction.pyx.

I think libc doesn't count as a part of the standard library, but I could be wrong.